### PR TITLE
Add note about tmp folder config for jvm

### PR DESCRIPTION
**Pull request by mosabua**
Fixes https://github.com/prestosql/presto/issues/2837

### CODE REVIEW

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 1]
```diff
@@ -92,6 +92,10 @@ Because an ``OutOfMemoryError`` typically leaves the JVM in an
 inconsistent state, we write a heap dump, for debugging, and forcibly
 terminate the process when this occurs.
 
+Some security standards restrict the ``/tmp`` folder usage. This can prevent
```
Comment (electrum) [inline presto-docs/src/main/sphinx/installation/deployment.rst]: We should be more specific and discuss noexec on Linux. The temporary folder needs to be on a file system without noexec.

**Comment by mosabua** [comment]
Good to go @electrum ?

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 2]
```diff
@@ -92,6 +92,11 @@ Because an ``OutOfMemoryError`` typically leaves the JVM in an
...
```
Comment (electrum) [inline presto-docs/src/main/sphinx/installation/deployment.rst]: Let's rephrase a bit
```
The temporary directory used by the JVM must allow execution of code. Specifically,
the mount must not have the ``noexec`` flag. The default ``/tmp`` directory is
mounted with this flag in some installations, which will prevent Presto from
starting. You can workaround this by overriding the temporary directory by adding
``-Djava.io.tmpdir=/path/to/other/tmpdir`` to the list of JVM options.

Comment (mosabua) [reply presto-docs/src/main/sphinx/installation/deployment.rst]: Applied this with removal of future usage

**Pull request merged**